### PR TITLE
AC_Autotune: remove most GCS messages

### PR DESCRIPTION
**Pull request by IamPete1**
Extract from https://github.com/ArduPilot/ardupilot/pull/15267, this removes the majority of autotune messages. We now just get the axis and stage

image

Current master is rather spammy:

image

All the information removed form the GSC messages is still in the `ATUN` and `ATDE` log messages at a higher rate.

### CODE REVIEW

**Comment by IamPete1** [comment]
Nice flash saving on MatekF405
```
Binary Name      Text [B]          Data [B]     BSS (B)      Total Flash Change [B] (%)      Flash Free After PR (B)
---------------  ----------------  -----------  -----------  ----------------------------  -------------------------
arducopter-heli  0 (0.0000%)       0 (0.0000%)  0 (0.0000%)  0 (0.0000%)                                       57324
blimp            0 (0.0000%)       0 (0.0000%)  0 (0.0000%)  0 (0.0000%)                                      305352
antennatracker   0 (0.0000%)       0 (0.0000%)  0 (0.0000%)  0 (0.0000%)                                      323992
ardusub          0 (0.0000%)       0 (0.0000%)  0 (0.0000%)  0 (0.0000%)                                      164656
arducopter       -1064 (-0.1134%)  0 (0.0000%)  0 (0.0000%)  -1064 (-0.1132%)                                  44028
arduplane        -1064 (-0.1088%)  0 (0.0000%)  0 (0.0000%)  -1064 (-0.1086%)                                   4536
ardurover        0 (0.0000%)       0 (0.0000%)  0 (0.0000%)  0 (0.0000%)                                      146752
```

File: libraries/AC_AutoTune/AC_AutoTune.cpp
Comment (khancyr) [inline libraries/AC_AutoTune/AC_AutoTune.cpp:244]: I think this one is interesting to keep

File: libraries/AC_AutoTune/AC_AutoTune.cpp
Comment (khancyr) [inline libraries/AC_AutoTune/AC_AutoTune.cpp:241]: this is less clear. 
IHMO we should just have a counter to the global axis with want to tune to follow the progression. There it is just a bunch of letter and number without meaning for most user

File: libraries/AC_AutoTune/AC_AutoTune.cpp
Comment (IamPete1) [reply libraries/AC_AutoTune/AC_AutoTune.cpp:244]: We have another pilot override warning with separate timer

https://github.com/ArduPilot/ardupilot/blob/adff48c3ea332acece4445f50f6c1300df2dad3f/libraries/AC_AutoTune/AC_AutoTune.cpp#L387-L392

**Comment by tridge** [comment]
@lthall I'm happy with this change, but needs your OK
also ping @rmackay9

**Comment by rmackay9** [comment]
I agree master is too spammy but I think this goes a bit too far.

I think a percentage would be more useful than the "1/4" and it would be good to keep the stage visible (e.g. roll/pitch + rate/angle + P/D).

I think removing the "Twitch" and "Leveling" messages is OK though.

**Comment by IamPete1** [comment]
I have re-worked to a little more of a middle ground.

image

**Review by khancyr — APPROVED** [review]
much better !

**Comment by IamPete1** [comment]
still a quite respectable flash saving. 

```
Binary Name      Text [B]          Data [B]     BSS (B)       Total Flash Change [B] (%)      Flash Free After PR (B)
---------------  ----------------  -----------  ------------  ----------------------------  -------------------------
arducopter-heli  0 (0.0000%)       0 (0.0000%)  0 (0.0000%)   0 (0.0000%)                                       57332
blimp            0 (0.0000%)       0 (0.0000%)  0 (0.0000%)   0 (0.0000%)                                      305552
antennatracker   0 (0.0000%)       0 (0.0000%)  0 (0.0000%)   0 (0.0000%)                                      324200
ardusub          0 (0.0000%)       0 (0.0000%)  0 (0.0000%)   0 (0.0000%)                                      164848
arducopter       -996 (-0.1062%)   0 (0.0000%)  4 (+0.0031%)  -996 (-0.1059%)                                   43964
arduplane        -1000 (-0.1023%)  0 (0.0000%)  0 (0.0000%)   -1000 (-0.1021%)                                   4432
ardurover        0 (0.0000%)       0 (0.0000%)  0 (0.0000%)   0 (0.0000%)                                      146760
```

**Comment by rmackay9** [comment]
Yes, much better, thanks!  Saves flash and better info for the user.

I'll leave it for @lthall to make the decision.

I guess one question is whether it still prints the final gains (when it successfully finds them) or not..  Ideally I think it should.

**Comment by IamPete1** [comment]
@rmackay9 No printing of the final gains yet, that will be the next PR.

**Review by lthall — APPROVED** [review]
I am happy for these messages to be simplified or removed.

**Comment by bnsgeyer** [comment]
@IamPete1 @rmackay9 can this wait until Tradheli autotune is merged. I have a massive set of changes.  Also the changes to the messages could be standardized across platforms.

**Comment by IamPete1** [comment]
Rebased, copter operates as above.

Heli is mostly unchanged:

- `AutoTune: Leveling (Angle(R) 5 > 4)` becomes `AutoTune: Leveling`
- `AutoTune: Start Test` is gone
- `AutoTune: (R) Rate D Up` becomes `AutoTune: Roll Rate D Up`

For example:

image

**Comment by rmackay9** [comment]
@bnsgeyer are you happy with this change?

I'm happy with the Copter change as we discussed before.

**Review by bnsgeyer — APPROVED** [review]
LGTM.

**Comment by rmackay9** [comment]
ok then, merged!

**Pull request merged**
